ssl/ssl_conf.c

```diff
@@ -94,6 +94,10 @@ struct ssl_conf_ctx_st {
     int *min_version;
     /* Pointer to SSL or SSL_CTX max_version field or NULL if none */
     int *max_version;
+    /* Default version mask for cipher_list if none provided, will be set to
+     * all versions up to TLSv1.2
```

Comment:
I didn't read the patch enough to know exactly what this means, but it sounds like it's either making the default only TLS 1.3 or disallowing TLS 1.3 by default, both of which sound weird.

Reply:
Yes, it sounds weird, but let me explain it, it's for backwards compatibility. Suppose following configuration:
`MaxProtocol = TLSv1.3`
`CipherString = AES`

This will automatically enable all 3 defaults ciphersuites from TLSv1.3 (like in current version).

When using these improved cipher list with version mask set to `ALL`, the same configuration in TLSv1.3 will enable only ciphersuites using AES. That means, 4 ciphersuites using AES will be enabled and ChaCha20 will be disabled. Because if at least one valid TLSv1.3 ciphersuite is in the `cipherlist`, or parameter `version_mask` is used, parameter `ciphersuites` will be set to empty string (all ciphers from them will be  erased)

Yesterday i pushed documentation, so you can read more information about that. These improvements should'n replace `ciphersuites` interface (or should?), but provide alternative way how to specify TLSv1.3 ciphersuites. 

So if we agree on, that we don't need backwards compatibility, i can make default all versions or e.g. only  TLS 1.2 and TLS 1.3.
Or will be better some option for seting default values at compile time?